content/kubevirt/index.md

```diff
@@ -158,13 +160,64 @@ podman push ${REGISTRY}/cnv/iis:latest
 oc apply -f https://raw.githubusercontent.com/openshift-examples/web/master/content/kubevirt/iis-template.yaml
 ```
 
+## Resource Capacity Calculation
+
+It’s the same as any other OpenShift sizing. Each VM has a request for the full amount of memory plus overhead. **There is no memory overcommitment (yet)**. Memory will be your biggest constraint and is how the cluster/nodes will be sized for the majority of deployments.
```

Comment:
Are you sure with the memory overcommitment? It might be available in 4.17 or 4.18?

Reply:
Sorry, my fault! I was of the opinion that the source from where I had it from was up2date. MemOvercommitment is supported. https://www.redhat.com/en/blog/whats-new-openshift-virtualization-417